### PR TITLE
Migration to MUI5

### LINKED ISSUE #69: Migrate to MUI5

### PR DESCRIPTION
**Pull request by swouf**
Close #69

### CODE REVIEW

File: src/components/modes/student/StudentView.js [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
 import Canvas from '../../main/Canvas';
 
-const styles = () => ({});
-
 const StudentView = () => <Canvas />;
 
-const StyledComponent = withStyles(styles)(StudentView);
-
-export default withTranslation()(StyledComponent);
+export default withTranslation()(StudentView);
```
Comment (spaenleh) [inline src/components/modes/student/StudentView.js:7]: I think if you use `useTranslation` hook inside all your functional components and you don't have any class components you can remove this, as it is useful to pass the `t` and `i18n` instances as props to the component.

File: src/components/modes/teacher/ImageUpload.js [hunk 2]
```diff
@@ -12,42 +13,22 @@ import { MAX_FILE_SIZE } from '../../../config/settings';
...
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  width: '100%',
+  marginTop: 2,
+  marginBottom: 1,
+}));
+
+const CloseButton = styled(Button)(() => ({
+  width: '20%',
+  fontSize: '1vw',
```
Comment (spaenleh) [inline src/components/modes/teacher/ImageUpload.js:27]: I am not sure you want to set your fontSize depending on the `viewWidth` (`vw`)  of the viewport. See [this article](https://www.freecodecamp.org/news/css-unit-guide/) for some alternatives.

File: src/components/modes/teacher/Settings.js [hunk 3]
```diff
@@ -74,37 +62,32 @@ const Settings = () => {
...
+        <StyledModal open={modalOpen} onClose={handleModalClose}>
+          <ModalContainer data-cy={SETTINGS_CY}>
+            <Typography sx={{ fontSize: '1.5vw' }}>{t('Settings')}</Typography>
             <ImageUpload />
             <BackgroundToggle />
             <DownloadActions />
-            <Divider className={classes.divider} />
-            <div className={classes.buttonContainer}>
-              <Button
+            <Divider sx={{ mt: 2, mb: 2 }} />
```
Comment (spaenleh) [inline src/components/modes/teacher/Settings.js:79]: `mt: 2, mb: 2` can be abreviated to `my: 2` (setting the margin on the y-axis using the short-hand property name)

File: src/components/modes/teacher/TeacherView.js [hunk 4]
```diff
@@ -1,12 +1,7 @@
...
 import Canvas from '../../main/Canvas';
 
-const styles = () => ({});
-
 const TeacherView = () => <Canvas />;
 
-const StyledComponent = withStyles(styles)(TeacherView);
-
-export default withTranslation()(StyledComponent);
+export default withTranslation()(TeacherView);
```
Comment (spaenleh) [inline src/components/modes/teacher/TeacherView.js:7]: Same as before, if ou are using `useTranslation` hook this is useless

**Review by spaenleh — CHANGES_REQUESTED** [review]
🎍Looks good, a few comments, and i am not sure that you want to use `vw` as units for your fontSize, but we can discuss.

File: src/components/main/CanvasToolbar.js [hunk 5]
```diff
@@ -67,15 +42,16 @@ const CanvasToolbar = () => {
...
-    </div>
+    <Stack
+      direction="column"
+      justifyContent="center"
+      spacing={2}
+      sx={{ position: 'fixed', bottom: 2, top: 2, right: 2 }}
+    >
+      <ToolButton onClick={deleteNote} disabled={disabled}>
+        <DeleteIcon />
+      </ToolButton>
```
[the remark this answers is not in the document]
Comment (swouf) [reply src/components/main/CanvasToolbar.js:53]: Yes, I changed it.

File: src/components/main/CanvasScaleControl.js [hunk 6]
```diff
@@ -1,42 +1,36 @@
...
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    maxWidth: '20%',
-    width: '20%',
-    maxHeight: '20%',
-  },
-  sliderStyle: {
-    width: '80%',
-  },
+const MainContainer = styled('div')(() => ({
```
[the remark this answers is not in the document]
Comment (swouf) [reply src/components/main/CanvasScaleControl.js:8]: Yes, I will change that I think. It is however not something that I will do immediately.

**Review by spaenleh — APPROVED** [review]
✨📝 Looks good to me ! Looking forward to using this new version of the app !

**Pull request merged**